dashboard/src/app/workspaces/share-workspace/user-item/user-item.html

```diff
@@ -19,16 +20,16 @@
            class="che-list-item-name">
         <span class="che-xs-header noselect" hide-gt-xs>Email</span>
         <span><img class="user-face" gravatar-src="userItemController.user.email"></span>
-        <span class="user-email che-hover ">{{userItemController.user.email}}</span>
+        <span class="user-email che-hover " name="member-name">{{userItemController.user.email}}</span>
       </div>
       <div flex-gt-xs="35">
         <span class="che-xs-header noselect" hide-gt-xs>Permissions</span>
-        <span class="user-permissions">{{userItemController.getUserActions()}}</span>
+        <span class="user-permissions" name="member-permissions">{{userItemController.getUserActions()}}</span>
```

Comment:
It seems is about _user_, not _member_.

Reply:
`name` attribute is usually used in **input** elements. It's better to use `id` for **div** element.

Reply:
Each member has permissions and **id** is not suitable here.